presto-hive/src/test/java/com/facebook/presto/hive/AbstractTestHiveClient.java

```diff
@@ -876,6 +879,23 @@ public void testPartitionSchemaMismatch()
     }
 
     @Test
+    public void testPartitionSchemaNonCanonical()
+            throws Exception
+    {
+        ConnectorTableHandle table = getTableHandle(tablePartitionSchemaChangeNonCanonical);
+        ConnectorColumnHandle column = null;
+        for (Map.Entry<String, ConnectorColumnHandle> entry : metadata.getColumnHandles(table).entrySet()) {
+            if (entry.getKey().equals("t_boolean")) {
+                column = entry.getValue();
+            }
+        }
+        assertNotNull(column);
+        ConnectorPartitionResult partitionResult = splitManager.getPartitions(table, TupleDomain.withFixedValues(ImmutableMap.of(column, false)));
+        assertEquals(partitionResult.getPartitions().size(), 1, "Expected 1 partition");
```

Comment:
These "Expected ..." messages duplicate the default messages (and worse, they might not match the actual assertion)